scripts/flattener.py

```diff
@@ -697,12 +680,93 @@ def check_matrix(m):
 	return m
 
 
+# If spatial transcriptomics, add adata.uns['spatial'] fields and eventually add full res
+def process_spatial():
+	global cxg_adata_raw
+	global cxg_obs
+	global cxg_obsm
+	global cxg_uns
+	global mfinal_adata
+	global mfinal_obj
+
+	if cxg_obs['assay_ontology_term_id'].unique()[0] in ['EFO:0010961','EFO:0030062']:
```

Comment:
Is this condition needed? Seems like the `if mfinal_obj['assays'] == ['spatial transcriptomics']:` would be the gatekeeper & make this redundant

Reply:
That's true. Seems like not a very useful if-statement. I think if/once we get other spatial transcript transcriptomic data, then we'll just revist.